yarn-project/aztec-node/src/aztec-node/server.ts

```diff
@@ -311,7 +311,7 @@ export class AztecNodeService implements AztecNode {
    * @param leafIndex - The index of the leaf for which the sibling path is required.
    * @returns The sibling path for the leaf index.
    */
-  public async getDataTreePath(leafIndex: bigint): Promise<SiblingPath<typeof NOTE_HASH_TREE_HEIGHT>> {
+  public async getNoteHashTreePath(leafIndex: bigint): Promise<SiblingPath<typeof NOTE_HASH_TREE_HEIGHT>> {
```

Comment:
As @iAmMichaelConnor also mentioned, would probably be better to go to
```suggestion
  public async getNoteHashSiblingPath(leafIndex: bigint): Promise<SiblingPath<typeof NOTE_HASH_TREE_HEIGHT>> {
```

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/3117/commits/13b3f90ebaf71a91bf3fe0d7068135515dfd2078